<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1135</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QScrollArea" name="scroll">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>440</y>
      <width>1141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1139</width>
       <height>139</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="go_left">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>98</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="go_right">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>98</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>right</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scroll_button">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>380</y>
      <width>98</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>scroll</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
